### ISSUE TITLE
[STY Post Live] LH report shows improper use of "role=text"

### ISSUE THREAD

**Issue by chris-hinds** [bbc/simorgh#6705]
**Describe the bug**
Since launching gahuza and ukchina STY's on Simorgh we have noticed that our Lighthouse score for a11y has dropped by 5 points.

Analysing a recent Lighthouse report shows that a role attribute with value of `text` is invalid and is dropping our a11y score.

[According to the ARIA Spec](https://www.w3.org/TR/wai-aria-1.1/#role_definitions) a role of `text` is not valid

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to 'https://www.bbc.com/gahuza/amakuru-52618917' in chrome
2. Right click to Inspect the page and run a Lighthouse audit
3. View the warnings in the Accessibility audit

**Expected behaviour**
We should be using a valid ARI role attribute

**Screenshots**
![image](https://user-images.githubusercontent.com/14273378/83243996-a4c8a280-a196-11ea-8773-e3b6193de616.png)

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by chris-hinds** [comment]
Closing as this is actually a bug within Lighthouse that was raise over a year ago [here](https://github.com/dequelabs/axe-core/issues/1597)
